libewf/libewf_single_files.h

```diff
@@ -59,6 +59,18 @@ struct libewf_single_files
 	/* The single file entry tree
 	 */
 	libcdata_tree_node_t *root_file_entry_node;
+
+	/* The single file source tree
+	 */
+	libcdata_tree_node_t *root_file_source_node;
```

Comment:
these might be stored as "trees", they appear to be more like an arrays

Reply:
True - there will only be one level to the tree so it's essentially an array.